Podfile.lock

```diff
@@ -38,12 +37,12 @@ CHECKOUT OPTIONS:
     :git: https://github.com/Kentzo/ShortcutRecorder.git
 
 SPEC CHECKSUMS:
-  AppCenter: 76ecf7e959a68e36b0fd314e8dc5897660214423
+  AppCenter: 8832b158d17e54845c3b88a91076218f0b07174d
   LetsMove: fefe56bc7bc7fb7d37049e28a14f297961229fc5
   ShortcutRecorder: fdf620aca34101b0cba3b10fca815e0459254189
-  Sparkle: ea073ba9c69cee3ebaab8685ba1e732e14d249f7
+  Sparkle: 270cd27377bf04e9c128af06e3a22d0f572d6ee3
   SwiftyMarkdown: 033abc9dad9a37903f81aec5188cade436f40520
 
-PODFILE CHECKSUM: 78df0a34910c4c031a65ac3b58dc004c21b552d0
+PODFILE CHECKSUM: f7be5f3250126444c8ff49c41543c32894eae84c
 
-COCOAPODS: 1.9.1
+COCOAPODS: 1.10.2
```

Comment:
What was the motivation behind 2e0685bb where you bumped cocoapods? I try to not change versions of anything unless there is an established benefit to it

Reply:
The new version is always (almost) better than the old one. Nothing should break from this. 🤓

Reply:
Better how?

I recall having issues with upgrading cocoapods in the past, as the CI would then be desync'd. Now the project cocoapods is aligned with the CI version and unless there is a clear benefit, I'd rather not touch things

Reply:
To use the current/latest version of Cocoapods I think is self-evident and not worth mentioning here. And obviously this version also worked on Travis without any apparent problems.